docs/user/customization.md

```diff
@@ -12,16 +12,162 @@ The most simple customization is exposed by the installer as an interactive seri
+    * `noProxy` (optional string): A comma-separated list of domains and [CIDRs][cidr-notation] for which the proxy should not be used.
+* `pullSecret` (required string): The secret to use when pulling images.
+* `sshKey` (optional string): The public Secure Shell (SSH) key to provide access to instances.
+
+### IP networks
+
+IP networks are represented as strings using [Classless Inter-Domain Routing (CIDR) notation][cidr-notation] with a traditional IP address or network number, followed by the "/" (slash) character, followed by a decimal value between 0 and 32 that describes the number of significant bits.
+For example, 10.0.0.0/16 represents IP addresses 10.0.0.0 through 10.0.255.255.
+
+### Machine pools
+
+The following machine-pool properties are available:
+
+* `hyperthreading` (optional string): Determines the mode of hyperthreading that machines in the pool will utalize.
+    Valid values are `Enabled` (the default) and `Disabled`.
+* `name` (required string): The name of the machine pool.
+* `platform` (optional object): Platform-specific machine-pool configuration.
+    * `aws` (optional object): [AWS-specific properties](aws/customization.md#machine-pools).
+    * `azure` (optional object): [Azure-specific properties](azure/customization.md#machine-pools).
+    * `openstack` (optional object): [OpenStack-specific properties](openstack/customization.md#machine-pools).
+    * `vsphere` (optional object): [vSphere-specific properties](vsphere/customization.md#machine-pools).
+* `replicas` (optional integer): The machine count for the machine pool.
+
+### Examples
+
+While all complete `install-config.yaml` will contain platform-specific sections, the following example fragments demonstrate platform-agnostic options:
+
+### Additional trust bundle
+
+```yaml
+apiVersion: v1
+additionalTrustBundle: |
+  -----BEGIN CERTIFICATE-----
+  ...base-64-encoded, DER Certificate Authority cert...
+  -----END CERTIFICATE-----
+
+  -----BEGIN CERTIFICATE-----
+  ...base-64-encoded, DER Certificate Authority cert...
+  -----END CERTIFICATE-----
+baseDomain: example.com
+metadata:
+  name: test-cluster
+proxy:
+  httpsProxy:  "https://username:password@proxy.example.com:123"
+  httpProxy: "httph://username:password@proxy.example.com:123"
+  noProxy: ...
+platform: ...
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+```
 
-- [AWS][aws-customization]
-- [Azure][azure-customization]
-- [OpenStack][openstack-customization]
+### Custom machine pools
+
+An example install config with custom machine pools to grow the size of the worker pool and disable hyperthreading:
+
+```yaml
+apiVersion: v1
+baseDomain: example.com
+controlPlane:
+  name: master
+  hyperthreading: Disabled
+compute:
+- name: worker
+  hyperthreading: Disabled
+  replicas: 5
+metadata:
+  name: test-cluster
+proxy:
+  httpsProxy:  "https://username:password@proxy.example.com:123"
+  httpProxy: "httph://username:password@proxy.example.com:123"
```

Comment:
@wking `httph`?

Reply:
> `httph`?

:man_facepalming: Fixed in 211796a93 -> 91c8e95b2.